modules/aws-eks-managed-node-groups/locals.tf

```diff
@@ -107,15 +107,30 @@ locals {
   #    local.managed_node_group["additional_iam_policies"]
   #  )) : []
 
+  k8s_labels = merge(
+    local.managed_node_group["k8s_labels"],
+    {
+      "eks/node_group_name" = local.managed_node_group["node_group_name"]
+    }
+  )
+
   common_tags = merge(
     var.context.tags,
     local.managed_node_group["additional_tags"],
     {
-      Name = "${var.context.eks_cluster_id}-${local.managed_node_group["node_group_name"]}"
-    },
-    {
+      Name                                                      = "${var.context.eks_cluster_id}-${local.managed_node_group["node_group_name"]}"
       "kubernetes.io/cluster/${var.context.eks_cluster_id}"     = "owned"
       "k8s.io/cluster-autoscaler/${var.context.eks_cluster_id}" = "owned"
       "k8s.io/cluster-autoscaler/enabled"                       = "TRUE"
   })
+
+  # NOTE: To support node groups to scale down to zero nodes,
+  # cluster autoscaler needs to more details to identify which nodes to scale-down.
+  # Here's the open issue and recommendations in GitHub: https://github.com/aws/containers-roadmap/issues/724
+  asg_tags = merge(
```

Comment:
instead of baking this in here, we should move it out to the example. the way its written currently means its forcing these tags on users, even if they don't use cluster autoscaler

Reply:
removed